pages/windows/del.md

```diff
@@ -0,0 +1,31 @@
+# del
+
+> Delete one or more files.
+
+- Delete one or more space-separated files or patterns:
+
+`del {{file_pattern}}`
+
+- Prompt for confirmation before deleting each file:
+
+`del {{file_pattern}} /p`
+
+- Force the deletion of read-only files:
```

Comment:
Why read-only ? Does it mean it won't delete writable files ?

Reply:
By default, `del` only removes files that aren't set to `Read-only` on the file system (there's a checkbox in the file explorer).

Using `/f` will remove any files that have the `Read-only` status set, as well as other files.

Reply:
I see. Got it.

Reply:
Ping @agnivade. Is this ok? 🤔